packages/cardpay-sdk/sdk/utils/general-utils.ts

```diff
@@ -381,3 +387,26 @@ export async function wait(ms = 1000) {
     setTimeout(resolve, ms);
   });
 }
+
+// Returns the current standard gas price in wei in the specified gas token
+export async function currentGasPrice(network: string, gasTokenAddress: string) {
+  let gasStationResponse = await fetch(`${getConstantByNetwork('relayServiceURL', network)}/v1/gas-station/`, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+      Accept: 'application/json',
+    },
+  });
+
+  let gasPriceInNativeToken = (await gasStationResponse.json()).standard;
```

Comment:
these are in units of `gwei` not `gwei`. I'm not seeing where you are doing a `gwei` to `wei` conversion (that will result in a number that is off by 9 decimal points!)

Reply:
No, I think this is in `wei`. If you check the response from https://relay-goerli.staging.stack.cards/api/v1/gas-station/, you can see the current value for standard is `2896112618`. If that was in gwei, then the gas price would be 2+ ethers which is improbable. Will document this and change naming so it is clear on which units we are operating